packages/core/.storybook/preview.js

```diff
@@ -32,7 +32,18 @@ export const parameters = {
   passArgsFirst: true,
   docs: {
     transformSource: (_src, storyContext) => {
-      return storyContext.originalStoryFn;
+      // cleanup story source code format
+      let storySource = storyContext.originalStoryFn.toString().split('\n');
+
+      // remove function wrapper
+      storySource.shift();
+      storySource.pop();
+      storySource[0] = storySource[0].replace('return html`', '').trim();
+      storySource[storySource.length - 1] = storySource[storySource.length - 1].replace('`;', '');
+
+      // shift tab space indent
+      storySource = storySource.map((line, i) => (i !== 0 ? line.split('').slice(2).join('') : line));
+      return storySource.join('\n');
```

Comment:
This is not great formatting but it removes the excess whitespace and function name around the code blocks without introducing a formatter dependency into the browser